src/workflows/recipes/components/RecipeDetails.js

```diff
@@ -282,15 +285,22 @@ export class ArgumentsValue extends React.PureComponent {
       valueRender = this.renderPreferencesTable;
     }
 
-    let textToCopy = value === undefined ? '' : value.toString();
-    if (this.compareInstances(value, [List, Map])) {
-      textToCopy = ArgumentsValue.stringifyImmutable(value);
+    // It's not good enough to check if the value is falsy to determine whether to show
+    // the default value instead. For example if it's "false" (type boolean) we want to
+    // display that as code. E.g. `<code>false</code>` or `<code>0</code>`.
+    let realValue = !(value === null || value === undefined);
+    let textToCopy = null;
+    if (realValue) {
+      textToCopy = value === undefined ? '' : value.toString();
+      if (this.compareInstances(value, [List, Map])) {
+        textToCopy = ArgumentsValue.stringifyImmutable(value);
+      }
     }
 
     return (
       <dd className={argumentsValueClassName}>
-        <div className="value">{valueRender(value)}</div>
-        {value ? (
+        <div className="value">{realValue ? valueRender(value) : defaultValue}</div>
+        {realValue ? (
```

Comment:
nit: this (and the `Tooltip` etc.) should probably be indented one level deeper

Reply:
I didn't write this. Prettier did. I'm not sure I want to attempt to override it.

Reply:
ahh!